EIPS/eip-5164.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 5164
+title: Cross-Chain Execution
+description: This specification defines an interface that supports execution across EVM networks.
+author: Brendan Asselstine (@asselstine), Anna Carroll (@anna-carroll), Hadrien Croubois (@Amxx), Nam Chu Hoai (@nambrot), Georgios (@geogons), Theo Gonella (@mintcloud), Rafael Solari (@rsolari), Auryn Macmillan (@auryn-macmillan), Nathan Ginnever (@nginnever)
+discussions-to: https://ethereum-magicians.org/t/eip-5164-cross-chain-execution/9658
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-14
+---
+
+## Abstract
+
+This specification defines a cross-chain execution interface for EVM-based blockchains. Developers will be able to have contracts call contracts on other chains. There are two parts to this spec: the "sending" interface and the "receiving" interface. The "sending" interface specifies the interface to send a call to another chain; this interface must be implemented by bridges. The "receiving" interface must be implemented by the developer; it is an interface for receiving cross-chain calls. This specification is agnostic of the bridge implementation; it simply standardizes the call interfaces.
```

Comment:
Swapped "developers" for "applications". I suspect bridges also have developers :wink: 

```suggestion
This specification defines a cross-chain execution interface for EVM-based blockchains. Developers will be able to have contracts call contracts on other chains. There are two parts to this spec: the "sending" interface and the "receiving" interface. The "sending" interface specifies the interface to send a call to another chain; this interface must be implemented by bridges. The "receiving" interface must be implemented by the application; it is an interface for receiving cross-chain calls. This specification is agnostic of the bridge implementation; it simply standardizes the call interfaces.
```

Reply:
Fixed in this commit: https://github.com/ethereum/EIPs/pull/5164/commits/da5558d00a3b5ab768df5b7d7336d92bd1f1c085#diff-41efe0617f861cc7330b025ecc86d55fe777d3c503208d83dca04258de4f5f41R15